app_runner.rb

```diff
@@ -0,0 +1,59 @@
+require_relative 'app'
+
+class AppRunner
+  def initialize
+    @app = App.new
+  end
+
+  def run
+    puts 'Welcome to Library App!'
+
+    loop do
+      display_options
+      choice = gets.chomp.to_i
+      break if choice == 7
+
+      handle_choice(choice)
+    end
+
+    puts 'Exiting...'
+  end
+
+  private
+
+  def display_options
+    puts 'Please choose an option:'
+    puts '1. List all books.'
+    puts '2. List all people.'
+    puts '3. Create a person (teacher or student, not a plain Person).'
+    puts '4. Create a book.'
```

Comment:
- [ ] It would be better if you fix this issue that occurs when creating a book
  
  ![image](https://github.com/theresetuyi/OOP-school-library/assets/73854999/bd3f07e7-e30c-42fb-a1ce-f335e75ccdb0)